spec/integration/release-build/after/Frameworks/a/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("@build_bazel_rules_ios//rules:app.bzl", "ios_application")
+load("@build_bazel_rules_ios//rules:framework.bzl", "apple_framework")
+load("@build_bazel_rules_ios//rules:test.bzl", "ios_unit_test")
+
+apple_framework(
+    name = "A",
+    srcs = glob([
+        "Sources/**/*.h",
+        "Sources/**/*.m",
+        "Sources/**/*.swift",
+    ]),
+    platforms = {"ios": "10.0"},
+    private_headers = glob(["Sources/Internal/**/*.h"]),
+    swift_version = "5.2",
+    visibility = ["//visibility:public"],
+)
+
+ios_unit_test(
+    name = "A-Unit-Tests",
+    srcs = glob([
+        "Tests/**/*.h",
+        "Tests/**/*.m",
+        "Tests/**/*.swift",
+    ]),
+    minimum_os_version = "10.0",
+    module_name = "A_Unit_Tests",
+    swift_version = "5.2",
+    deps = [":A"],
+)
+
+ios_application(
+    name = "A-App",
+    srcs = glob([
+        "App/**/*.h",
+        "App/**/*.m",
+        "App/**/*.swift",
+    ]),
+    bundle_id = "org.cocoapods.A-App",
+    families = [
+        "iphone",
+        "ipad",
+    ],
+    minimum_os_version = "10.0",
+    module_name = "A_App",
+    swift_version = "5.2",
+    xcconfig = {
```

Comment:
The `after` BUILD file here would look similar to the one I mentioned before. Leveraging `xcconfig_by_build_setting `